bioblend/galaxy/objects/wrappers.py

```diff
@@ -1250,6 +1251,21 @@ def run(self, inputs, history, wait=False,
         return outputs
 
 
+class Job(Wrapper):
+    """
+    Maps to a Galaxy job.
+    """
+    BASE_ATTRS = Wrapper.BASE_ATTRS
+    POLLING_INTERVAL = 10  # for output state monitoring
```

Comment:
@nsoranzo Which line?? BASE_ATTRS and/or POLLING_INTERVAL? I added them to keep consistency with other Wrappers. And for future developments.